exercise-book/src/self-check.md

```diff
@@ -33,8 +35,8 @@ In this exercise we will create a small in-memory message queue that is accessib
 
 1. Create a Cargo workspace for your project.
 2. Create a binary package inside your workspace for your TCP server
-3. Implement a simple TCP Server that listens for connections on `127.0.0.1:7878`. For each incoming connection, read all of the input as a string, and send it back to the client.
-4. Create a library package inside your workspace for a message protocol parser. Make your TCP server depend on that library using a relative path.
+3. Implement a simple TCP Server that listens for connections on `127.0.0.1:7878`. For each incoming connection, read all of the input as a string, and send it back to the client. Disconnect the client if they send input that is not valid UTF-8.
+4. Create a package with a library crate inside your workspace for the message protocol parser. Make your TCP server depend on that library using a relative path.
 5. Inside your library implement the following function so that it implements the protocol specifications to parse the messages. Use the provided tests to help you with the case handling.
 
     ```rust, ignore
```

Comment:
This one keeps the comma?

Reply:
Good point. The comma breaks syntax highlighting in Github, and is ignored by mdbook, so it should be removed.